### PR TITLE
Previews: remove dashboard previews backend

### PR DESCRIPTION
**Pull request by ArturWierzbicki**
part of https://github.com/grafana/grafana/issues/65693

### CODE REVIEW

**Review by torkelo — APPROVED** [review]

**Review by chri2547 — APPROVED** [review]
Docs LGTM! Thank you for the update!

**Review by ryantxu — APPROVED** [review]
😢 but as an external app it will be great

File: pkg/services/sqlstore/migrations/dashboard_thumbs_mig.go [hunk 1]
```diff
@@ -1,31 +1,31 @@
 package migrations
 
-import "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+// This migration was run behind the `dashboardPreviews` flag. The feature flag and the whole dashboard previews feature was removed in https://github.com/grafana/grafana/pull/66176
```
Comment (bergquist) [inline pkg/services/sqlstore/migrations/dashboard_thumbs_mig.go:3]: I would remove these migrations completely since they are independent anyway.

**Review by bergquist — APPROVED** [review]
Made a comment about the migrations but LGTM regardless :)

File: pkg/services/sqlstore/migrations/dashboard_thumbs_mig.go (hunk 1 above)
Comment (DanCech) [reply pkg/services/sqlstore/migrations/dashboard_thumbs_mig.go:3]: I was concerned that having records in the migrations table that refer to migrations that don't exist might be problematic but it seems that the migrator doesn't actually care about that.  If anything we might want to add a migration to drop the table if it does exist.

Comment (ArturWierzbicki) [reply pkg/services/sqlstore/migrations/dashboard_thumbs_mig.go]: thanks both - in the next PR ill either remove this file, or add a migration to remove the table if thats easy enough

**Pull request merged**